.github/workflows/nightly-int-test-workflow.yml

```diff
@@ -27,13 +38,14 @@ jobs:
       PRIVATE_REGISTRY: ${{ secrets.ECR_REPOSITORY }}
       S3_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
       ENTERPRISE_LICENSE_LOCATION: ${{ secrets.ENTERPRISE_LICENSE_LOCATION }}
+      CLUSTER_WIDE: "true"
     steps:
       - name: Set Test Cluster Name
         run: |
           echo "TEST_CLUSTER_NAME=eks-integration-test-cluster-${{ matrix.test }}-$GITHUB_RUN_ID" >> $GITHUB_ENV
       - name: Set Test Cluster Nodes and Parallel Runs
         run: >-
-          if [ ${{ matrix.test }} == *"appframework"* ]; then 
+          if grep -q "appframework" <<< "${{ matrix.test }}"; then
             echo "CLUSTER_WORKERS=5" >> $GITHUB_ENV
```

Comment:
I thought we use 3 cluster workers

Reply:
Not adding anything new. Its existing approach for appframework test as they are more resource heavy.